tools/build.sh

```diff
@@ -84,8 +95,39 @@ case "${LANGUAGE}" in
     ;;
 esac
 
+if [[ -n "${SECTION_SEARCH}" ]]; then
+  TARGET=$(grep "include{" structure.tex | grep "$SECTION_SEARCH")
+
+  # Target is empty
+  if [[ -z "$TARGET" ]]; then
+    echo "Error: No section found matching '$SECTION_SEARCH'" >&2
+    exit 1
+  fi
+
+  # Target is ambiguous
+  if [[ $(echo "$TARGET" | wc -l) -gt 1 ]]; then
+    echo "Error: Multiple sections found matching '$SECTION_SEARCH':" >&2
+    echo "$TARGET" >&2
+    exit 1
+  fi
+
+  echo "$TARGET" > structure.tex
+fi
+
+cleanup() {
+  if [[ -n "${SECTION_SEARCH}" ]]; then
+    git restore structure.tex
+  fi
+  if [[ ${LANGUAGE} != en ]]; then
+    git restore po4a.cfg
+  fi
+}
+
+trap cleanup EXIT
+
 if [[ ${LANGUAGE} != en ]]; then
   # limit output to specified language
+  sed -i "s/pl es fr ru ua de cs he/${LANGUAGE}/" po4a.cfg
```

Comment:
I think it's better to replace the line with `po4a_langs`, because otherwise when new language is added, we'll forget to update this list